README.md

```diff
@@ -1,59 +1,99 @@
 # Thali Cordova Plugin
 
-<Work on progress, not near release quality !!>
-## Goals
-The Thali Cordova Plugin is an easy-to-use Cordova plugin for building peer-to-peer (P2P) networking
-apps on Android and iOS. 
+This project is a work in progress and not yet production-level quality.
 
-The Thali Cordova Plugin is layered on the JXcore Cordova plugin, which uses JXcore to allow one to build
-mobile applicatons in JavaScript for Node.JS.  
+The Thali Cordova Plugin is a [Cordova](http://cordova.apache.org/) plugin for building peer-to-peer (P2P) networking apps on Android and iOS.
+
+The Thali Cordova Plugin is layered on the [JXcore Cordova plugin](https://github.com/jxcore/jxcore-cordova), which uses [JXcore](http://jxcore.com/home/) to allow one to build mobile applications in JavaScript for Node.JS.  
 
 ## Prerequisites
 
-* For Android, [Android Studio](http://developer.android.com/sdk/index.html)
-* For iOS, [Xcode 6](https://developer.apple.com/xcode/), or later
+### Android
+
+Download [Android Studio](http://developer.android.com/sdk/index.html)
+
+Make sure to set your `ANDROID_HOME` environment variable:
+
+Mac OS X (put in your `~/.bash_profile` file):
+```
+export ANDROID_HOME=~/Library/Android/sdk
+export PATH=${PATH}:$ANDROID_HOME/tools:$ANDROID_HOME/platform-tools
+```
+
+Linux (put in your `~/.bashrc` file):
+```
+export ANDROID_HOME=/<installation location>/Android/sdk
+export PATH=${PATH}:$ANDROID_HOME/tools:$ANDROID_HOME/platform-tools
+```
+
+Windows:
+```
+set ANDROID_HOME=C:\<installation location>\Android\sdk
+set PATH=%PATH%;%ANDROID_HOME%\tools;%ANDROID_HOME%\platform-tools
+```
+
+### iOS
+
+Download [Xcode 6](https://developer.apple.com/xcode/), or later.
 
 ## Getting Started
 
 ### Install latest Node.JS
+
 Follow the instructions at [https://nodejs.org/](https://nodejs.org/). When you're done, check that the
-installation worked: 
+installation worked:
 ```
-~>   node -v
-v0.12.2
-~>   npm -v
-2.7.4
+$ node -v
```

Comment:
Is there a reason why we require node.js (v0.12) as opposed to just JXCore? They will need JXCore eventually and JXCore is 100% backwards compatible with Node.js so why not just use JXCore?